src/webgui/src/webgui/image_pub.py

```diff
@@ -23,10 +23,10 @@ def publish_images(self):
 			self.image_pub.publish(msg_frame)
 
 def main(args):
-	if (len(args) == 1):
+	if (len(args) < 4):
```

Comment:
I think a better way to handle parameters like these is rosparam: http://wiki.ros.org/rospy/Overview/Parameter%20Server.

This lets us configure it in our launch files easily.

Reply:
The node name (as well as the file name) should be different each time we run it (can't have two nodes with same name). I am under the impression that rospy params are configuration settings, not something you would change every time the file is executed. Can rospy params be easily changed?

Reply:
I think it may be a good thing not to be able to run two copies of this node. If we have 2 of them running, the second one will crash anyways because the port it wants is already bound.

Regarding rosparam, we would have the ability to set parameters both on the command line and in the launch files